android/src/main/kotlin/com/onegini/mobile/sdk/flutter/handlers/CustomRegistrationHandler.kt

```diff
@@ -0,0 +1,49 @@
+package com.onegini.mobile.sdk.flutter.handlers
+
+import com.onegini.mobile.sdk.flutter.providers.CustomRegistrationAction
+import java.lang.Exception
+
+class CustomRegistrationHandler {
+    fun actionRespondSuccess(
+        idProvider: String?,
+        customRegistrationActions: ArrayList<CustomRegistrationAction>,
+        token: String?) {
+        val action = getCustomRegistrationAction(idProvider, customRegistrationActions)
```

Comment:
You can change to
```
fun actionRespondSuccess(
        idProvider: String?,
        customRegistrationActions: ArrayList<CustomRegistrationAction>,
        token: String?) = getCustomRegistrationAction(idProvider, customRegistrationActions)
   .?let { action.returnSuccess(token) } ?: return 
```
+ add ticket link in the TODO comment.

Reply:
reworked this part, unsure if this is still relevant